index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <title>Portfolio</title>
+</head>
+<body>
+   
+        <!---Header-->
+        <header class="tool-bar">
+            <a href="#">Welcome</a>
+            <nav class="menu"><i class="fas fa-bars"></i></nav>
+        </header>
+           
+        <div class="container">
```

Comment:
Please use the `section` tag as the Headline section container.

Reply:
Thanks for the review.I'll do accordingly.